e2etest/newe2e_resource_manager_interface.go

```diff
@@ -24,6 +24,8 @@ type RemoteURIOpts struct {
 type AzureURIOpts struct {
 	// Must be manually specified
 	WithSAS bool
+	// Location level for SAS token, defaults to the resource's location level.
+	Level cmd.LocationLevel
```

Comment:
Unused?

Reply:
this is intended to be optionally set by devs using the test framework to adjust Level if they need to.

Reply:
irrelevant now